src/compiler/proposals-and-stabilization.md

```diff
@@ -285,14 +285,25 @@ circumstance.
   - Open an RFC describing the motivation for the change and start an FCP to approve, start an FCP.
   - If approved, the change should be accompanied by a blog post announcing the change with a
     notice period of at least one release before the change applies.
-- Demoting/removing a target
+- Demoting/removing a Tier 1 target
+  - **Propose using:** RFC
+  - **Approve using:** FCP
+  - Demoting a Tier 1 target requires a RFC as per the
+    [Tier 1 Target Policy][tier-1-target-policy]. Consult the
+    [Tier 1 Target Policy][tier-1-target-policy] for approval and community
+    communication requirements.
+- Demoting/removing a Tier 2 target
   - **Propose using:** MCP
   - **Approve using:** FCP
-  - Write an MCP describing why the target should be demoted/removed and once discussion has
+  - Write an MCP describing why the Tier 2 target should be demoted/removed and once discussion has
```

Comment:
Note: I think this is not symmetrical: promoting a Tier 2 target seems to only require {MCP, Second} not FCP, e.g.

- https://github.com/rust-lang/compiler-team/issues/761

Reply:
I don't think we need to lean too much on past precedent since there's likely a bunch of conflicting examples in the past, we should just decide on something reasonable and stick with that